charts/rs-server-catalog/values.yaml

```diff
@@ -171,3 +171,10 @@ stac_browser:
 tempo:
   # -- Grafana tempo endpoint.
   endpoint: http://grafana-tempo-distributor.logging.svc.cluster.local:4317
+
+# -- OpenTelemetry
```

Comment:
You can remove the special comment string `# --` for the `otel:` line because it makes redundant values in the readme :
```markdown
| otel | object | `{"trace_body":false,"trace_headers":false}` | OpenTelemetry |
| otel.trace_body | bool | `false` | Trace request bodies and response contents with OpenTelemetry ? |
| otel.trace_headers | bool | `false` | Trace request headers with OpenTelemetry ? |
```